src/common/marlin_vars.h

```diff
@@ -37,25 +37,31 @@ enum {
     MARLIN_VAR_TIMTOEND = 0x1d,            // R:  uint32, oProgressData.oTime2End.mGetValue() or -1 if not valid
     MARLIN_VAR_FAN0_RPM = 0x1e,            // R:  uint16, fanctl0.getActualRPM()
     MARLIN_VAR_FAN1_RPM = 0x1f,            // R:  uint16, fanctl1.getActualRPM()
-    MARLIN_VAR_FAN_CHECK_ENABLED = 0x20,   //RW: uintt8, fan_check
-    MARLIN_VAR_FS_AUTOLOAD_ENABLED = 0x21, //RW: uint8_t fs_autoload
-    MARLIN_VAR_MAX = MARLIN_VAR_FS_AUTOLOAD_ENABLED
+    MARLIN_VAR_FAN_CHECK_ENABLED = 0x20,   // RW: uintt8, fan_check
+    MARLIN_VAR_FS_AUTOLOAD_ENABLED = 0x21, // RW: uint8_t fs_autoload
+    MARLIN_VAR_CURR_POS_X = 0x22,          // R: float current_position
```

Comment:
R or RW?
why is X different from Y Z E?

Reply:
👍